ansible/tasks/delivery/delivery.yaml

```diff
@@ -0,0 +1,91 @@
+---
+- name: Provision a persistent volume for Kafka
```

Comment:
Just a small thing here. I think the folder should be named provision, as the whole structure would be ansible/tasks/provision/delivery.yaml and it would hold both delivery and publishing yamls.